src/framework/renderer/js/RendererUtilities.js

```diff
@@ -307,16 +307,17 @@ fluid_3_0_0 = fluid_3_0_0 || {};
     fluid.renderer.selection.inputs = function (options, container, key, config) {
         fluid.expect("Selection to inputs expander", options, ["selectID", "inputID", "labelID", "rowID"]);
         var selection = config.expander(options.tree);
+        var expandedOpts = config.expandLight(options);
```

Comment:
There are some docs on the docs site for this at http://docs.fluidproject.org/infusion/development/RendererComponentTreeExpanders.html#selection-to-inputs-expander